.github/workflows/ci.yml

```diff
@@ -235,16 +241,17 @@ jobs:
         name: Download frontend coverage
         uses: actions/download-artifact@v3
         with:
-          name: jest-coverage-report
-          path: src-ui/
+          path: src-ui/coverage/
       -
         name: Upload frontend coverage to Codecov
         uses: codecov/codecov-action@v3
         with:
           # not required for public repos, but intermittently fails otherwise
           token: ${{ secrets.CODECOV_TOKEN }}
           flags: frontend
-          directory: src-ui/
+          directory: src-ui/coverage/
+          # dont include backend coverage files here
+          files: '!coverage.xml'
```

Comment:
So even with the directory setting, it still finds the backend coverage somehow?  That almost sounds like a bug in their action.

Reply:
Yea I played with it a bit. 

No this is 'our' fault. In the download step above I download all of the artifacts (no name filter included) and they all get put into this directory (including docs artifact and backend tests report).

The alternative is to have 4 download steps with hard-coded names of each jest shard report.

I thought this was a bit more elegant (you can change num shards in one place and this would be fine) but lmk if you feel otherwise.